### PR TITLE
Add detection prediction tutorial

### PR DESCRIPTION
**Pull request by Louis-Dupont**
I made sure to include answers of the following issues:
- get bbox information - https://github.com/Deci-AI/super-gradients/issues/930
- get bbox information - https://github.com/Deci-AI/super-gradients/issues/907
- fps & filtering - https://github.com/Deci-AI/super-gradients/issues/892
- set_processing_params: https://github.com/Deci-AI/super-gradients/issues/903

Let me know if you feel like I'm still missing important questions/explanations or if it's just not clear.

**TODO: update the collab before merging**

### CODE REVIEW

File: documentation/source/ModelPredictions.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+]
+
+images_predictions = model.predict(IMAGES)
+```
+You can use the default IoU and Confidence threshold or override them like this:
+
+```python
+images_predictions = model.predict(IMAGES, iou=0.5, conf=0.7)
+```
+- `iou`: IoU threshold for the non-maximum suppression (NMS) algorithm. If None, the default value associated with training is used.
```
Comment (shaydeci) [inline documentation/source/ModelPredictions.md:42]: Thats (currently) not true, its defined by the model class.

File: documentation/source/ModelPredictions.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
+# Using Pretrained Models for Predictions 
+
+In this tutorial, we will demonstrate how to use the `model.predict()` method for object detection tasks.
+
+The model used in this tutorial is [YOLO-NAS](YoloNASQuickstart.md), pre-trained on the [COCO dataset](https://cocodataset.org/#home), which contains 80 object categories.
+
+
+**Warning**: If your model was trained on your own custom dataset, make sure to follow these [steps first](DetectionPredictionCustom.md).
```
Comment (shaydeci) [inline documentation/source/ModelPredictions.md:8]: Its a bit unclear...
A custom dataaset can also be one that uses our COCO Format Detection Dataset class.
In that case (and actually in any case where we just trian with SG objects and use our transforms for detection) there is nothing to be done (recall my feature that takes the validation transforms and creates a pipeline from it).

Please also mention that, as in that case you save the users trouble defining these things.

**Review by shaydeci — CHANGES_REQUESTED** [review]

File: documentation/source/ModelPredictions.md (hunk 2 above)
Comment (Louis-Dupont) [reply documentation/source/ModelPredictions.md]: 35af53371555093a4ab5aea8bb73120296bb8ee6
I refered to SG datasets, and not transforms explicitly because the SG detection datasets all work with SG transforms, so I think there is no point to go too deep in implementations details:
What I wrote;
You are using SG dataset ? -> No need to set the params
You are not ? -> set the params

File: documentation/source/ModelPredictions.md (hunk 1 above)
Comment (Louis-Dupont) [reply documentation/source/ModelPredictions.md]: https://github.com/Deci-AI/super-gradients/commit/35af53371555093a4ab5aea8bb73120296bb8ee6

**Review by shaydeci — APPROVED** [review]
LGTM
